ecl/data_source_ecl_network_fic_gateway_v2_test.go

```diff
@@ -0,0 +1,253 @@
+package ecl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccNetworkV2FICGatewayDataSource_basic(t *testing.T) {
```

Comment:
以下のように属性名に変更していただけますか？
```suggestion
func TestAccNetworkV2FICGatewayDataSource_name(t *testing.T) {
```